infogami/utils/template.py

```diff
@@ -100,7 +105,7 @@ def find(path):
 
 #@@ Find a better name
 class Render(storage.DictPile):
```

Comment:
I think the errors on the python3 tests : `TypeError: Can't instantiate abstract class Render with abstract methods __delitem__, __iter__, __len__, __setitem__` imply that these methods need to be implemented on the `Render()` class.

I think I may have found a rough solution to this problem in an older (discarded) PR, if I remember correctly @tfmorris gave some feedback on it... I'll try to find the changes and see how they relate to this.

Reply:
This is what I was thinking of:

https://github.com/internetarchive/infogami/pull/9/files#diff-a9c5443a321681b08daeb14f84d30163R22-R25

which is the same changes.

And the discussion I remembered was from a similar change but in a different file:
https://github.com/internetarchive/infogami/pull/9/files#diff-6d1be1eecf8f7b3d481304b45030a594R209-R227

Pasting the links here, I'll try to make sense of the differences soon.